pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -50,6 +51,9 @@ var (
 	// while debouncing. Defaults to 10 seconds. If events keep
 	// showing up with no break for this time, we'll trigger a push.
 	DebounceMax time.Duration
+
+	// Statically link protobuf descriptors from UDPA
+	_ = udpa.TypedStruct{}
```

Comment:
but why? What does importing this do? How does  `_ = udpa.TypedStruct{}` in any way impact the code?

Reply:
It calls this function https://github.com/cncf/udpa/blob/master/go/udpa/type/v1/typed_struct.pb.go#L73